nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/main/java/org/nuxeo/elasticsearch/commands/IndexingCommandsStacker.java

```diff
@@ -70,9 +72,22 @@ public void stackCommand(DocumentEventContext docCtx, String eventId) {
             return;
         }
         if ("/".equals(doc.getPathAsString())) {
-            log.debug("Skip indexing command for root document");
-            return;
+            if (log.isDebugEnabled()) {
```

Comment:
You don't need to test if log is debug enabled because we don't construct the string.